posthog/tasks/test/test_email.py

```diff
@@ -58,8 +58,9 @@ def test_send_invite(self, MockEmailMessage: MagicMock) -> None:
         send_invite(invite.id)
 
         assert len(mocked_email_messages) == 1
-        assert mocked_email_messages[0].send.call_count == 1
-        assert mocked_email_messages[0].html_body
+        the_message = mocked_email_messages[0]
```

Comment:
Hmm I'd say the better change to run backend CI here would be to update `ci-backend.yml` to run on any `frontend/public/email/` change